src/App.js

```diff
@@ -8,15 +8,16 @@ import React from "react";
 
 const { setConfig, getValue } = Config;
 
-const testApiRoot = "https://structuredcontentdev.bcg.ad.bcgov.us/api/news";
-const prodApiRoot = "https://services.baltimorecountymd.gov/hub/api/news";
+const testApiRoot =
+  "https://structuredcontentdev.bcg.ad.bcgov.us/api/hub/news/news";
+const prodApiRoot = "https://services.baltimorecountymd.gov/api/hub/news/news";
```

Comment:
this link doesn't look right based on your changes.

`api/hub/structuredContent/news` is what i'm guessing it would be.